app/src/main/res/layout/fragment_workshop_weapon_filter_body.xml

```diff
@@ -275,7 +275,16 @@
                     android:id="@+id/toggle_blast"
                     style="@style/ButtonToggle.ImageOnly"
                     app:imageSize="@dimen/image_size_large"
-                    app:src="@drawable/ic_status_blast" />
+                    android:layout_marginEnd="@dimen/margin_medium"
+                    app:src="@drawable/ic_status_blast">
+
+                </com.gatheringhallstudios.mhworlddatabase.components.CheckedImageButton>
+
+                <com.gatheringhallstudios.mhworlddatabase.components.CheckedImageButton
+                    android:id="@+id/toggle_non_elemental"
+                    style="@style/ButtonToggle.ImageOnly"
+                    app:imageSize="@dimen/image_size_large"
+                    app:src="@drawable/ic_question_mark" />
```

Comment:
Use this icons instead. https://github.com/gatheringhallstudios/MHWorldDatabase/blob/master/app/src/main/res/drawable-anydpi/ic_ui_armor_missing.xml